internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -85,6 +96,15 @@ func (e *EnvStackConfig) Template() (string, error) {
 		}
 		mft = string(out)
 	}
+
+	forceUpdateID := e.PrevForceUpdateID
+	if e.in.ForceUpdate {
+		id, err := uuid.NewRandom()
```

Comment:
💭 random thoughts - no action item:

I was thinking if we need to have a way of making this ID more understandable like using the git commit SHA so that they can figure out what caused this template change. But I realized that we have `env show --manifest` now ✅ so they can always figure what caused the template change